src/lumigo_opentelemetry/instrumentations/django/tested_versions/3.10/django

```diff
@@ -1 +1,60 @@
-4.2.5
\ No newline at end of file
+3.2
```

Comment:
added these versions to ensure that the problem is resolved

Reply:
Didn't understand why...

Reply:
yep 100%

Reply:
as discussed out of band, every push, the tracer testing is done and the testing is performed on all the instrumentation files - ie the tested versions files. so when i add all those versions to django's tested versions files, they're going to be tested in the tracer testing. it would be nice if i could just add one or two versions, but until we're confidently merging the version testing PRs i don't want the README to have two or three random versions, it looks weird.